### ISSUE TITLE
add a "max" toggle in the pay screen dialogue 

### ISSUE THREAD

**Issue by Roasbeef** [lightninglabs/lightning-app#865]
Once this PR is merged into `lnd` it will be possible to instruct lnd to sweep all on chain coins to a single address. This can be useful if one wishes to switch wallets, or simply withdrawal all the funds in an existing wallet. The main [proto modification is the addition of a `send_all` flag](https://github.com/lightningnetwork/lnd/pull/2198/files#diff-f6ec8c0f52caccfaec5656028d316c7fR841). If this is set (and no amount is set) then we'll sweep all the coins into a single address. This was possible before, but only via trial and error as one had to guess the total fees (based on the number of UTXOs in the wallet) as the magic number to input for the send logic. Additionally, even if their guess was very close, they'd likely end up with a very small output that may not even be economical to spend as left over. 

We can integrate this new feature in the form of a "max" toggle in the pay screen dialogue. This is related to #863 as without that, if you sweep all the funds to the same address within lnd, then they'd all be used again to open channels.

**Comment by tanx** [comment]
Also related to #656

**Comment by tanx** [comment]
@Roasbeef good call. Definitely should add this for the mainnet release so users can wipe their funds after disabling autopilot in the settings. Suggested with something like what Coinbase does to Lee and @bryanvu ...

![image](https://user-images.githubusercontent.com/1374174/54802164-15c41e80-4ca5-11e9-91a8-2f97f6334707.png)

**Comment by tanx** [comment]
@valentinewallace turns out Lee already created something like a `MAX` toggle for the SLS screens. He suggested we just reuse it in the `PayBitcoinView` screen so users can flush their funds after deactivating autopilot and closing all channels:

![image](https://user-images.githubusercontent.com/1374174/54898575-cc273e00-4f07-11e9-97a0-973d8f4e9c0d.png)

**Comment by valentinewallace** [comment]
@tanx that's great!

The problem is that `lnd` isn't able to `estimateFee` for withdrawing all coins (it errors saying the amount is too high).

The current solution idea I have is to decrement the amount until `estimateFee` doesn't fail. Let me know if you have a better idea, otherwise consider this a heads up for why the code is the way it is :sweat_smile:
